packages/augur-ui/src/modules/reporting/common.tsx

```diff
@@ -588,52 +610,76 @@ export class ReportingBondsView extends Component<
       reportingGasFee,
       reportAction,
-      preFilledStake,
+      inputtedReportingStake,
       userAvailableRep,
       id,
+      migrateRep,
+      migrateMarket,
+      initialReport,
     } = this.props;
 
     const { showInput, disabled, scalarError, stakeError, isScalar } = this.state;
-    const preFilled = preFilledStake || '0';
+    const preFilled = inputtedReportingStake || '0';
 
+    const repAmount = migrateRep ? formatRep(preFilled).formatted : formatAttoRep(market.noShowBondAmount).formatted;
     // id === "null" means blank scalar, user can input new scalar value to dispute
     return (
       <div
         className={classNames(Styles.ReportingBondsView, {
           [Styles.Scalar]: isScalar,
+          [Styles.InitialReport]: initialReport,
+          [Styles.MigrateRep]: migrateRep,
+          [Styles.MigrateMarket]: migrateMarket,
         })}
       >
-        {isScalar && id === "null" && (
+        {isScalar && id === 'null' && (
           <ScalarOutcomeView
             inputScalarOutcome={inputScalarOutcome}
             updateScalarOutcome={this.updateScalarOutcome}
             scalarDenomination={market.scalarDenomination}
             scalarError={scalarError}
           />
         )}
-        {market.reportingState === REPORTING_STATE.OPEN_REPORTING && (
-          <>
-            <span>Review Initial Reporting</span>
-            <LinearPropertyLabel
-              key="initial"
-              label="open reporter winning Stake"
-              value={`${formatAttoRep(market.noShowBondAmount).formatted} REP`}
-            />
-          </>
+        {migrateRep && (
+          <InputRepStake
+            stakeAmount={String(inputtedReportingStake)}
+            updateStakeAmount={this.updateInputtedStake}
+            stakeError={stakeError}
+            max={String(userAvailableRep)}
+            maxLabel='MAX'
+          />
         )}
+        <span>Review</span>
+        <LinearPropertyLabel
+          key="initial"
+          label={migrateRep ? 'REP to migrate' : 'open reporter winning Stake'}
+          value={`${repAmount} REP`}
+        />
         <LinearPropertyLabel
           key="totalEstimatedGasFee"
           label="Transaction Fee"
           value={reportingGasFee}
         />
-        <PreFilledStake
-          showInput={showInput}
-          toggleInput={this.toggleInput}
-          updatePreFilledStake={this.updatePreFilledStake}
-          preFilledStake={preFilledStake}
-          stakeError={stakeError}
-          threshold={userAvailableRep}
-        />
+        {migrateRep && createBigNumber(inputtedReportingStake).lt(createBigNumber(userAvailableRep)) &&
+          <DismissableNotice
+            show={true}
+            description=''
+            title='Are you sure you only want to migrate a portion of your REP to this universe?
+            If not, Go back and select the ‘MAx’ button underneath the amount input to migrate your Full REP Amount.'
```

Comment:
Nit - capitalization: `If not, go back and select the ‘Max’ button underneath the amount input to migrate your full REP amount.`

Reply:
good one, will fix